src/main/java/net/sf/jabref/gui/importer/fetcher/EntryFetchers.java

```diff
@@ -30,13 +31,19 @@ public EntryFetchers(JournalAbbreviationLoader abbreviationLoader) {
         entryFetchers.add(new DOAJFetcher());
         entryFetchers.add(new SpringerFetcher());
 
-        entryFetchers.add(new IdBasedEntryFetcher(new DiVA(Globals.prefs.getImportFormatPreferences())));
-        entryFetchers.add(new IdBasedEntryFetcher(new IsbnFetcher(Globals.prefs.getImportFormatPreferences())));
         entryFetchers.add(new SearchBasedEntryFetcher(new ArXiv()));
         entryFetchers.add(new SearchBasedEntryFetcher(new GvkFetcher()));
     }
 
     public List<EntryFetcher> getEntryFetchers() {
         return Collections.unmodifiableList(this.entryFetchers);
     }
+
+    public static List<IdBasedFetcher> getIdFetchers() {
+        List<IdBasedFetcher> list = new LinkedList<>();
```

Comment:
Why not `ArrayList`? It is [faster at sorting](http://stackoverflow.com/a/26013151/873282).